docs/antrea-network-policy.md

```diff
@@ -667,6 +684,14 @@ If set with a `podSelector`, all matching Pods from Namespaces selected by the
 A ClusterGroup with `ipBlock` referenced in an ACNP's `appliedTo` field will be
 ignored, and the policy will have no effect.
 
+**serviceReference**: Pods that serve as the backend for the specified Service
+will be grouped. Services without selectors are currently not supported, and will
+be ignored if referred to in ClusterGroup serviceReference.
+When ClusterGroups with serviceReference are used in ACNPs as `appliedTo` or
+`to`/`from` peers, no Service port information will be automatically assumed for
+traffic enforcement. ServiceReference is merely used for ClusterGroups to keep
+in sync with a Service on how to group Pods.
```

Comment:
For the last sentence: ServiceReference is merely a mechanism to group Pods and ensure that a ClusterGroup stays in sync with the set of Pods selected by a given Service.

Reply:
Done